### PR TITLE
CF-1101: Concorde message schema

### PR DESCRIPTION
**Pull request by scadams**

### CODE REVIEW

File: sample_product_schemas/concorde-message.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <attribute name="emailSentTime" type="dateTime" use="optional">
+        Time the email was sent, if successful.
+    </attribute>
+    <attribute name="errorMessage" type="string" use="optional">
+        Error message encountered from the external system when trying to send this email, if any.
+    </attribute>
+    <attribute name="errorDescription" type="string" use="optional">
+        Internal description of an error when trying to send this email, if any.
+    </attribute>
+    <attribute name="namespace" type="name" use="required" allowedValues="FLASHMOB MEANDER">
```
Comment (shintasmith) [inline sample_product_schemas/concorde-message.xml]: I think the type needs to be Name with upper case N

Comment (scadams) [inline sample_product_schemas/concorde-message.xml]: Updated. I wonder if that should be specified here? http://docs-internal.rackspace.com/cloud-feeds/api/v1.0/feeds-publishers-guide/content/Allowed_types.html

File: sample_product_schemas/concorde-message.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<productSchema xmlns="http://docs.rackspace.com/core/usage/schema"
+               namespace="http://docs.rackspace.com/event/concorde/message"
+               serviceCode="Concorde"
```
Comment (scadams) [inline sample_product_schemas/concorde-message.xml]: Unsure if this serviceCode is appropriate.

File: sample_product_schemas/concorde-message.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+               version="1" resourceTypes="CONCORDE_MESSAGE"
```
Comment (scadams) [inline sample_product_schemas/concorde-message.xml]: Unsure if we need to specify resourceTypes or leave it out, or if the name is appropriate if we leave it in.

File: sample_product_schemas/concorde-message.xml (hunk 2 above)
Comment (shintasmith) [inline sample_product_schemas/concorde-message.xml]: serviceCode is quite correct. It is just a name to identify your team/product.

File: sample_product_schemas/concorde-message.xml (hunk 3 above)
Comment (shintasmith) [inline sample_product_schemas/concorde-message.xml]: I think you should add resourceTypes. If this event is to describe Concorde Message, then it is the correct resourceType to use.

**Comment by shintasmith** [comment]
I have updated the source of the wrong documentation, wrt 'name' vs 'Name'. The docs will get rebuilt when our doc person comes back from vacation this week.

The rest looks good. But would you mind following the instructions here:
http://docs-internal.rackspace.com/cloud-feeds/api/v1.0/feeds-publishers-guide/content/Using_Cloud_Feeds.html

You will need to do a build, generate some samples, etc.

I have another PR that handles step 4.3 from the instructions above, so you can skip that one.

**Comment by scadams** [comment]
Alrighty, updated with everything you requested.

**Comment by scadams** [comment]
`mvn test` also passes.

**Comment by scadams** [comment]
Sigh... I missed the part where you said you already handled step 4.3. If it's alright, can I just leave it in this PR? I believe I needed to add it here in order to fully test everything, right? If not, I can take it out in favor of your PR.

**Comment by shintasmith** [comment]
@scadams, this PR looks good! Thanks for doing the whole thing. I would just close my PR that does only 4.3.

**Comment by shintasmith** [comment]
ok to test

**Pull request merged**